ansible_base/jwt_consumer/common/auth.py

```diff
@@ -296,7 +296,12 @@ def process_rbac_permissions(self):
 
             for index in object_indexes:
                 object_data = self.token['objects'][object_type][index]
-                resource, obj = self.get_or_create_resource(object_type, object_data)
+                try:
+                    resource, obj = self.get_or_create_resource(object_type, object_data)
+                except IntegrityError:
+                    logger.warning(f"Got integrity error on {object_data}. Skipping {object_type} assignment.")
```

Comment:
Also, if I saw this I'd be like why is this happening, how do I fix this? Should we add something like "This will be resolved in the next periodic sync" or something along those lines?